isis/src/base/apps/topds4/topds4.xml

```diff
@@ -57,5 +57,33 @@
         </description>
       </parameter>
     </group>
+
+    <group name="Other">
+      <parameter name="MAINLABEL">
+        <type>boolean</type>
+        <default><item>true</item></default>
+        <brief>
+          Use the main cube label as input data
+        </brief>
+        <description>
+          Use the main input cube label as a source of data to replace template elements with.
+        </description>
+      </parameter>
+
+      <parameter name="ORIGINALLABEL">
+        <type>boolean</type>
+        <default><item>false</item></default>
+        <brief>
+          Use the original label as input data
+        </brief>
+        <description>
+          Use the original label attached to the input cube as a source of data 
+          to replace template elements with. This is the label that was saved when
+          the cube was first ingested into ISIS as a cube.
+        </description>
+      </parameter>

```

Comment:
These don't need to be here. We can always dump the main label to the data and then dump the original label if it's there. That way, users can just specify it if they want it in the template and not have to worry about setting the parameters and the template correctly.